aip/0135.md

```diff
@@ -179,6 +179,16 @@ message DeleteBookRequest {
 If the etag is provided and does not match the server-computed etag, the
 request **must** fail with a `FAILED_PRECONDITION` error code.
 
+### Errors
+
+If the user does not have permission to access the resource, regardless of
+whether or not it exists, the service **must** error with `PERMISSION_DENIED`
+(HTTP 403). Permission **must** be checked prior to checking if the resource
+exists.
+
+If the user does have proper permission, but the requested resource does not
+exist, the service **must** error with `NOT_FOUND` (HTTP 404).
+
```

Comment:
Is it worth including the logic behind this and the "effective new meaning" of the 403?